lib/containers/container_images.pm

```diff
@@ -133,8 +127,8 @@ sub build_with_zypper_docker {
     # If zypper-docker list-updates lists no updates then derived image was successfully updated
     assert_script_run("zypper-docker list-updates $derived_image | grep 'No updates found'", 240);
 
-    my $local_images_list = script_output("$runtime image ls");
-    die("$runtime $derived_image not found") unless ($local_images_list =~ $derived_image);
+    my $local_images_list = script_output("$runtime->{runtime} image ls");
```

Comment:
We have `get_images_by_repo()` so why it isn't more universal e.g. `get_images()` with optimal parameters?

Reply:
same as the other comments

Reply:
I don't understand. Sorry man :(

Reply:
Can you let me provide a followup PR to address the remaining things. 
in this PR, i left intentionally some things untouched. the main point is to make all of the test use the runtime.pm. but to apply everything in once, it would required a huge amount of validations. i thought that letting things for later will simplify the review. So is it ok to leave it as it is for now? 
To be honest i do not know what can go wrong in this point but i know that i will need to run a few and if something do not work for some reason, it will cost some extra time, that i do not know if there is now.

Reply:
Agreed already long time ago.